src/components/util/wallet.tsx

```diff
@@ -8,7 +7,7 @@ export interface WalletDetails {
 }
 
 export const connectWallet = async (): Promise<providers.Web3Provider> => {
-  await window.ethereum.enable();
+  await window.ethereum.request({ method: "eth_requestAccounts" });
```

Comment:
something new ?

Reply:
window.ethereum.enable getting depreciated soon, they put in documentation to change to this 
https://docs.metamask.io/guide/ethereum-provider.html#legacy-methods